src/components/Input/Input.tsx

```diff
@@ -56,9 +79,21 @@ class Input extends UIComponent<any, any> {
 
   inputRef: any
 
-  computeTabIndex = props => {
-    if (!_.isNil(props.tabIndex)) return props.tabIndex
-    if (props.onClick) return 0
+  constructor(props, context) {
+    super(props, context)
+
+    this.state = {
+      value: props.value || '',
+    }
+  }
```

Comment:
Looks like we have a default prop here.  Also, no need for a constructor:

```jsx
state = { value: props.value }

static defaultProps = { value: '' }
```

This will ensure the default value is advertised.